### DISCUSSION TITLE
Two questions about the Physical greaseweazel device

### DISCUSSION THREAD

**Discussion by Jimbo7136** [keirf/greaseweazle#233] [Q&A] [answered]
1. Is it possible to boot from a floppy attached by the device?

2. Can it be used for drivers in installing windows 2000/XP etc?

Thank you.

**Comment by keirf** [comment, marked as answer]
No the greaseweazle cannot act as a native floppy controller. There are no OS drivers. It has to be accessed via special greaseweazle-compatible tools.
